tap.go

```diff
@@ -56,6 +56,7 @@ type FeedArguments struct {
 	KeysOnly   bool          // If true, client doesn't want values so server shouldn't send them.
 	Terminator chan bool     // Feed will be terminated when this channel is closed (DCP Only)
 	DoneChan   chan struct{} // DoneChan is closed when the mutation feed terminates.
+	GroupID	   string        // Config group ID the Sync Gateway instance is running with
```

Comment:
Everything else in FeedArguments is related to the internal DCP feed processing, which makes me wonder whether this is the right way to pass GroupID down to StartDCPFeed.  If it does, I wonder if it makes sense to name it here based on its usage inside the DCP Feed (i.e. define CheckpointPrefix here, instead of groupID).  What do you think?